team_members.md

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
not this file, but the `_team_members` directory is missing a `steve_saylor.md` file.

Reply:
actually, don't worry about this. i'll add my own.